### PR TITLE
Translations update from Lynix :3

### PR DESCRIPTION
**Pull request by ItsLynix**
Swedish added :)

### CODE REVIEW

File: src/main/resources/languages/sv_SE.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{    
+    "help_title": "Hjälp",
+    "help_commands_title": "Kommandon",
+    "help_commands_content": "Ogiltigt kommando. Giltiga kommandon: {commands}.",
+    "help_context_title": "Kontext",
+    "help_context_content": "Vissa kommandon är bara tillgängliga i  DMs,  medan andra bara är tillgängliga i servers.",
+
+    "error_generic": "Ett internt  fel uppstod.",
```
Comment (Derkades) [inline src/main/resources/languages/sv_SE.json:8]: Double space

File: src/main/resources/languages/sv_SE.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{    
+    "help_title": "Hjälp",
+    "help_commands_title": "Kommandon",
+    "help_commands_content": "Ogiltigt kommando. Giltiga kommandon: {commands}.",
+    "help_context_title": "Kontext",
+    "help_context_content": "Vissa kommandon är bara tillgängliga i  DMs,  medan andra bara är tillgängliga i servers.",
```
Comment (Derkades) [inline src/main/resources/languages/sv_SE.json:6]: Double space

File: src/main/resources/languages/sv_SE.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "ping_usage": "Användning: `{command} <discord server id>`",
+    "ping_working": "API-webbanslutningen verkar fungera. Tog {time} ms.",
+
+    "apiurl_usage": "Användning: `{command} <server id> <api url>`.",
+    "apiurl_success": "Din Discord-server har länkats till din NamelessMC-webbplats.",
+    "apiurl_url_invalid": "Den angivna API-URL: en ser inte ut som en NamelessMC API URL. Du hittar API-URL: en i StaffKP> Konfiguration> API.",
+    "apiurl_url_malformed": "Den angivna API-URL: en är felaktig.",
+    "apiurl_failed_connection": "Det gick inte att ansluta till din webbplats. Se meddelandet ovan för mer information.",
+    "apiurl_already_used": "Denna API-URL används redan av en Discord-server, använd `{command}` för att ta bort länken.",
+    "apiurl_try_https": "Din API-URL använder http: //. Om du använder en proxy för HTTPS visar webbplatsen fortfarande HTTP API-URL: er. Försök att köra det här kommandot igen och ersätt http: // med https: // i API-URL: n.",
```
Comment (Derkades) [inline src/main/resources/languages/sv_SE.json:32]: Are you sure about the unusual space after `http:`?

File: src/main/resources/languages/sv_SE.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "verify_token_invalid": "Det gick inte att verifiera ditt konto, du har angett en ogiltig token.",
+    "verify_not_used": "Det går inte att verifiera ditt konto, du tillhandahöll en verifieringstoken för en webbplats som inte använder den här botten.",
+    "verify_success": "Tack för att du verifierar, ditt konto är nu länkat.",
+
+    "ping_usage": "Användning: `{command} <discord server id>`",
+    "ping_working": "API-webbanslutningen verkar fungera. Tog {time} ms.",
+
+    "apiurl_usage": "Användning: `{command} <server id> <api url>`.",
+    "apiurl_success": "Din Discord-server har länkats till din NamelessMC-webbplats.",
+    "apiurl_url_invalid": "Den angivna API-URL: en ser inte ut som en NamelessMC API URL. Du hittar API-URL: en i StaffKP> Konfiguration> API.",
```
Comment (Derkades) [inline src/main/resources/languages/sv_SE.json:28]: Space before `>` for consistency please

**Review by Derkades — CHANGES_REQUESTED** [review]
Thanks! I don't speak Swedish so I can't comment much, but I did notice a few minor typos

**Comment by ItsLynix** [comment]
I wrote the double spaces with a broken keyboard on accident, and the space after http is not supposed to be there, I've been on vacation for almost a month, but I'm now back to correct this, thanks for telling me. Really shows the quality control on this stuff.

**Review by ItsLynix — COMMENTED** [review]
I've fixed the errors now..

**Review by Derkades — APPROVED** [review]
still a couple unfixed issues but I'll just fix those myself after merging

**Comment by ItsLynix** [comment]
Apologies for some of the issues, working with a broken keyboard isn't really too reliable in editing

**Pull request merged**